uuid_test.go

```diff
@@ -224,6 +224,33 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestBytes(t *testing.T) {
```

Comment:
This test is not actually required.  Instead, TestCoding should be modified to include a test of Bytes().

Reply:
done 😁